lib/workers/repository/update/branch/schedule.spec.ts

```diff
@@ -299,15 +299,17 @@ describe('workers/repository/update/branch/schedule', () => {
       });
     });
 
-    describe('complex cron schedules', () => {
+    describe('handles schedule with DOM and DOW using AND logic', () => {
```

Comment:
```suggestion
    describe('handles schedule with Day Of Month and Day Of Week using AND logic', () => {
```

Expanding the abbreviation is easier to read.